data/stoa0114a/stoa003/stoa0114a.stoa003.opp-lat1.xml

```diff
@@ -375,7 +367,7 @@
 </div>
 
 <div n="4" subtype="poem" type="textpart">
-<ab> <title>IIII.  EPITHALAMIVM DICTVM MAXIMO V. S. PRAEFATIO. </title> </ab>
+<ab> <title >IIII.  EPITHALAMIVM DICTVM MAXIMO V. S. PRAEFATIO. </title> </ab>
```

Comment:
A space seems to have crept into every `<title> ` tag

Reply:
Ah, that was my fault, from tryng to add something automatically and then not having looked through whole file. I will go fix that.

Reply:
And was that for all of the files @lcerrato or just this one?

Reply:
Actually I did a find across the files it was in two files. I've fixed it and will push the changes.

Reply:
hey @lcerrato I fixed the last issue with the space in the title element so can I merge this? Thanks!